js/google_ima.js

```diff
@@ -442,6 +443,13 @@ require("../html5-common/js/utils/utils.js");
           //displaying it, so just notify AMC that we are showing an overlay.
           else
           {
+            //provide width and height values if available. Alice will use these to resize
+            //the skin plugins div when a non linear overlay is on screen
+            if (this.currentAMCAdPod && this.currentNonLinearIMAAd)
+            {
+              this.currentAMCAdPod.width = this.currentNonLinearIMAAd.getWidth();
+              this.currentAMCAdPod.height = this.currentNonLinearIMAAd.getHeight();
+            }
             // raise WILL_PLAY_NONLINEAR_AD event and alert AMC and player that a nonlinear ad is started.
             // Nonlinear ad is rendered by IMA.
             _amc.sendURLToLoadAndPlayNonLinearAd(this.currentAMCAdPod, this.currentAMCAdPod.id, null);
```

Comment:
Is it ok to include the width and height with the "currentAMCAdPod" object here?